### PR TITLE
Updated packs

### PR DESCRIPTION
**Pull request by melenevskyi**
### Background

Updated packs

### CODE REVIEW

**Review by egibs — DISMISSED** [review]

File: packs/atlassian.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ Description: Group of all Atlassian detections
 PackDefinition:
   IDs:
     - Atlassian.User.LoggedInAsUser
+    - Confluence.0DayIPs
```
Comment (arielkr256) [inline packs/atlassian.yml:7]: These IOCs were last updated # 2022-06-02 Confluence 0-Day IOCs: This rule probably needs to be deprecated.

File: packs/onepassword.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ PackDefinition:
     - Standard.OnePassword.SignInAttempt
     # 1Password Specific Rules
     - OnePassword.Unusual.Client
+    - OnePassword.Lut.Sensitive.Item
+    - OnePassword.Sensitive.Item
+    - OnePassword.Login.From.CrowdStrike.Unmanaged.Device
```
Comment (arielkr256) [inline packs/onepassword.yml:13]: I like how rules that require multiple log sources are in both packs

File: packs/panther.yml [hunk 3]
```diff
@@ -7,6 +7,10 @@ PackDefinition:
     - Panther.SAML.Modified
     - Panther.Sensitive.Role
     - Panther.User.Modified
+    - IOC.SunburstFQDNIOCs
```
Comment (arielkr256) [inline packs/panther.yml:10]: All of these IOC/0DayIP rules need to be deprecated, they are pretty stale at this point.  Also the IOC.Log4jExploit rule I believe causes severe performance issues.

**Review by arielkr256 — CHANGES_REQUESTED** [review]
Deprecate IOC rules and remove from packs

**Review by le4ker — APPROVED** [review]

**Pull request merged**